src/chaplin/lib/utils.coffee

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+import mediator from '../mediator'
```

Comment:
No, the require was intended. There's a dependency cycle between utils and mediator.